### DISCUSSION TITLE
What should I return is right case when the type is void, Either<Error, void>

### DISCUSSION THREAD

**Discussion by MiniSuperDev** [gcanti/fp-ts#1814] [Q&A]
![image](https://user-images.githubusercontent.com/105388540/204098303-305d4bf8-8a32-4d10-ac39-fe4748cf4b38.png)
```ts
import * as E from 'fp-ts/lib/Either';

function fun(): E.Either<Error, void> {
  return E.right(void); // Expresion expected.
}

```

Or should I use other type instead of void?

thank you

**Comment by 0x706b** [comment]
`void` is a keyword that discards the value of the expression that follows it. `undefined` or `void 0` would work. However, `Option<Error>` might be a more simple way to model this type
